Replace initial notification with walkthroughs (#291)
VSpaceCode/VSpaceCode: walkthroughs/welcome/customization.md

# Customization

> You don't like the default bindings of the menu, you can change it to your liking however you want. Learn more in our [documentation](command:vspacecode.openDocumentationUrl).

Reviewer: ```suggestion
If you don't like the default bindings of the menu, you can change them to your liking. Learn more in our [documentation](command:vspacecode.openDocumentationUrl).
```
